documentation/c01-whatis.sil

```diff
@@ -82,11 +82,105 @@ SILE does not benefit from the large ecosystem and community that has grown up a
+}
+
+\item{\strong{Market usage:}
+	The Typst’s developers’ background in academia has heavily influenced how and where it is used.
+	The largest demographic is almost certainly university students writing papers for their math and science curriculum, with some journal publishing thrown in for good measure.
+	SILE’s active developers are book publishers and the majority of the work it has grown up around is literature and languages.
+	Its flexibility also makes it an ideal tinkerer’s tool: used in font proofing and other workflows unrelated to book publishing.
+}
+
+\item{\strong{Speed:}
+	Typst is faster than SILE.
+	Full stop.
+	This may matter a lot when you’re making lots of changes to detailed formulas and need to review previews.
+	This may matter less if you work on a book design first with minimal content, copy-edit the actual content separately, then just combine the two in a final step.}
+
+\item{\strong{Platform support:}
+	Typst’s pure Rust implementation with WASM-only plugins makes it easy to compile and run for many platforms including Linux, macOS, Windows, and even the web.
+	SILE’s approach to programming with system access does require a bit more “wiring” to get working on a platform, which currently limits it to use on Linux and macOS.
+	Windows support is certainly possible, although to date has not been published.}
+
+\end{itemize}
+
+Another alternative worth mentioning is Speedata Publisher.
+Speedata is a bit of a Unicorn: using much of LuaLaTeX under the hood while being neither a LaTeX engine nor a wrapper.
+Its main use case is taking XML data and laying it out programmatically; and the emphasis is on data handling.
+If you are trying to typeset a product catalog from a database or data-rich newsletters using fancy templates it might be worth a look.
+
+Most other alternatives are based in one way or another on browser technology.
+WeasyPrint and some others\footnote{PDFreactor, Prince, Antenna House, Typeset.sh, Vivliostyle, BFO Publisher, etc.} take HTML and CSS input and converts them to printable output using web-technology libraries.
+Paged.js and several others\footnote{pdfChip, quarkdown, etc.} do the same by actually shipping the content off to a full browser and asking for print versions back.
+Like paged.js, Quarkdown offloads the actual typesetting to a browser, but uses a home-grown superset of Markdown as its input format instead of HTML plus CSS directly.
+
+See \href{https://polytype.dev} for samples comparing and contrasting many typesetting engines.
+
 \section{SILE versus InDesign and competitors}
 
 The other category of tool that people reach for when designing printed material on a computer desktop publishing software (DTP).
-Adobe's InDesign is a prominent package in this space, but many others exist.
-Affinity Publisher is a newcommer but popular alternative.
+Adobe’s InDesign is a prominent package in this space, but many others exist.
+Affinity Publisher is a newcomer but popular alternative.
```

Comment:
What the point in name-dropping Affinity (and Scribus etc below) without comparison?

(I always thought the _only_ reason for mentioning InDesign was the SILE's frames (however executed) derived from an idea that was there.)